koku/masu/celery/tasks.py

```diff
@@ -312,3 +315,36 @@ def vacuum_schemas():
     for schema_name in schema_names:
         LOG.info("Scheduling VACUUM task for %s", schema_name)
         vacuum_schema.delay(schema_name)
+
+
+@app.task(name="masu.celery.tasks.clean_volume", queue_name="clean_volume")
+def clean_volume():
+    """Clean up the volume in the worker pod."""
+    LOG.info("Cleaning up the volume at %s " % Config.PVC_DIR)
+    # get the billing months to use below
+    months = DateAccessor().get_billing_months(Config.INITIAL_INGEST_NUM_MONTHS)
+    db_accessor = ReportManifestDBAccessor()
+    # this list is initialized with the .gitkeep file so that it does not get deleted
+    assembly_ids_to_exclude = [".gitkeep"]
```

Comment:
I am not really sure the context of the issue you are working on, but I was wondering if this was something that may be worth defaulting in the `masu/config.py`? So that way we could change assembly_ids we want to exclude on the fly in openshift. The answer could totally be no, but I figured it would be worth suggesting just in case. 🤷‍♂

Reply:
This list is actually populated below in line 332. However, I initialized it with `.gitkeep` inside because if you look at the repo (https://github.com/project-koku/koku/tree/master/testing/pvc_dir) the only thing in testing/pvc_dir is the `.gitkeep` file - the volume is mapped to our `koku/testing/pvc_dir` when you develop using docker. So if we were to run this locally it deletes all files that do not have one of the "assembly_ids" in them and this was my way of keeping the `.gitkeep` file from being deleted. I am open to doing this differently or renaming the list to make it more clear but I'm not sure that moving it `masu/config` is more clear. That being said I'm definitely open to other suggestions/ arguments for moving it there.

Reply:
Nah, I wasn't sure if it would be needed or not just thought I would through it out there and get your opinion on if it would be worth it on not. The whole volume mapping explained it pretty well. Thanks ash!